src/main/java/com/o19s/grandcentral/kubernetes/PodManager.java

```diff
@@ -343,7 +344,7 @@ else if (left.getLastRequest() < right.getLastRequest())
    * @throws IOException
    */
   private void refreshPods() throws IOException {
-    HttpGet podsGet = new HttpGet("https://" + k8sConfiguration.getMasterIp() + ":443/api/v1/namespaces/" + k8sConfiguration.getNamespace() + "/pods");
+    HttpGet podsGet = new HttpGet(k8sConfiguration.getProtocol() + "://" + k8sConfiguration.getMasterIp() + "/api/v1/namespaces/" + k8sConfiguration.getNamespace() + "/pods");
```

Comment:
I'd like to see the GKE dashboard again and verify they just surface up an IP. 

If we rename the parameter to `master_url` and include `https` in the sample config file we have the same number of parameters today and don't rely on a hidden configuration value.